src/client/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -230,7 +230,7 @@ public bool IsBrokerAvailable()
         /// in order to override the default authority set at the application construction. Note that the overriding authority needs to be part
         /// of the known authorities added to the application construction.
         /// </remarks>
-        [Obsolete("This method is deprecated. Please use AcquireTokenByUsernamePassword(IEnumerable<string> scopes, string username, string password) instead.", true)]
+        [Obsolete("This method is deprecated. Please use AcquireTokenByUsernamePassword(IEnumerable<string> scopes, string username, string password) instead.", false)]
```

Comment:
nit: I remember @pmaytak read some guidelines about warnings and messages and they recommend against using "please".

Reply:
Yea, I remember that too, although I can't find the related docs.